_pkgdown.yml

```diff
@@ -47,3 +47,7 @@ reference:
   - create_agg_tree
   - create_scale_tree
   - vis_tree
+
+- title: internal
```

Comment:
This came from here https://pkgdown.r-lib.org/reference/build_reference.html#missing-topics

There might be a better solution, I couldn't figure out how other R packages (tidyverse) handle this.